### PR TITLE
feat(atom/input): improved documentation

### PR DESCRIPTION
**Pull request by juanmaguitar**
- https://sui-components-improve-docs-atom-input.now.sh/workbench/atom/input/demo
- https://sui-components-improve-docs-atom-input.now.sh/workbench/atom/input/documentation/api
- https://sui-components-improve-docs-atom-input.now.sh/workbench/atom/input/documentation/readme

### CODE REVIEW

**Review by danivm — DISMISSED** [review]

**Review by danivm — APPROVED** [review]

File: components/atom/input/src/index.js [hunk 1]
```diff
@@ -28,64 +28,64 @@ const AtomInput = ({type, ...props}) => {
...
-  /* A hint to the user of what can be entered in the control. The placeholder text must not contain carriage returns or line-feeds. */
+  /** A hint to the user of what can be entered in the control. The placeholder text must not contain carriage returns or line-feeds. */
   placeholder: PropTypes.string,
 
-  /* This Boolean attribute prevents the user from interacting with the input */
+  /** This Boolean attribute prevents the user from interacting with the input */
   disabled: PropTypes.bool,
 
-  /* 's' or 'm', default: 'm' */
+  /** 's' or 'm', default: 'm' */
```
Comment (carlesnunez) [inline components/atom/input/src/index.js:76]: I'll remind here the importance on the usage of inputSizes static const's

File: components/atom/input/src/index.js [hunk 2]
```diff
@@ -28,64 +28,64 @@ const AtomInput = ({type, ...props}) => {
...
   size: PropTypes.oneOf(Object.values(inputSizes)),
 
-  /* width of input based in number of characters (native "size" attribute) */
+  /** width of input based in number of characters (native "size" attribute) */
   charsSize: PropTypes.number,
 
-  /* true = error, false = success, null = neutral */
+  /** true = error, false = success, null = neutral */
```
Comment (carlesnunez) [inline components/atom/input/src/index.js:82]: Cool!!!!! 🔝

**Review by carlesnunez — APPROVED** [review]
Small changes that doesn't must be done.

LGTM! Thank you so much for improve the README.md

File: components/atom/input/src/index.js (hunk 1 above)
Comment (juanmaguitar) [reply components/atom/input/src/index.js:76]: This is properly explained in the README

**Pull request merged**